examples/compareLambdaTypes.py

```diff
@@ -67,63 +70,64 @@ def setUp(self, CFHT=True):
         self.subconfig4.regularizationType = "forwardDifference"
         self.subconfig4.forwardRegularizationOrders = [1, 2]
 
-        
-
         self.kList1 = ipDiffim.makeKernelBasisList(self.subconfig1)
-        self.bskv1  = ipDiffim.BuildSingleKernelVisitorF(self.kList1, pexConfig.makePolicy(self.subconfig1))
+        self.bskv1 = ipDiffim.BuildSingleKernelVisitorF(self.kList1, pexConfig.makePolicy(self.subconfig1))
 
         self.kList2 = ipDiffim.makeKernelBasisList(self.subconfig2)
-        self.hMat2  = ipDiffim.makeRegularizationMatrix(pexConfig.makePolicy(self.subconfig2))
-        self.bskv2  = ipDiffim.BuildSingleKernelVisitorF(self.kList2, pexConfig.makePolicy(self.subconfig2), self.hMat2)
+        self.hMat2 = ipDiffim.makeRegularizationMatrix(pexConfig.makePolicy(self.subconfig2))
+        self.bskv2 = ipDiffim.BuildSingleKernelVisitorF(
+            self.kList2, pexConfig.makePolicy(self.subconfig2), self.hMat2)
 
         self.kList3 = ipDiffim.makeKernelBasisList(self.subconfig3)
-        self.hMat3  = ipDiffim.makeRegularizationMatrix(pexConfig.makePolicy(self.subconfig3))
-        self.bskv3  = ipDiffim.BuildSingleKernelVisitorF(self.kList3, pexConfig.makePolicy(self.subconfig3), self.hMat3)
+        self.hMat3 = ipDiffim.makeRegularizationMatrix(pexConfig.makePolicy(self.subconfig3))
+        self.bskv3 = ipDiffim.BuildSingleKernelVisitorF(
+            self.kList3, pexConfig.makePolicy(self.subconfig3), self.hMat3)
 
         self.kList4 = ipDiffim.makeKernelBasisList(self.subconfig4)
-        self.hMat4  = ipDiffim.makeRegularizationMatrix(pexConfig.makePolicy(self.subconfig4))
-        self.bskv4  = ipDiffim.BuildSingleKernelVisitorF(self.kList4, pexConfig.makePolicy(self.subconfig4), self.hMat4)
+        self.hMat4 = ipDiffim.makeRegularizationMatrix(pexConfig.makePolicy(self.subconfig4))
+        self.bskv4 = ipDiffim.BuildSingleKernelVisitorF(
+            self.kList4, pexConfig.makePolicy(self.subconfig4), self.hMat4)
 
         # known input images
         defDataDir = lsst.utils.getPackageDir('afwdata')
```

Comment:
In the spirit of not fixing examples as part of this ticket, you can ignore this comment.